rdkit/Chem/fmcs/fmcs.py

```diff
@@ -1171,8 +1170,7 @@ def canon(cangen_nodes):
 def get_closure_label(bond_smarts, closure):
   if closure < 10:
     return bond_smarts + str(closure)
-  else:
-    return bond_smarts + "%%%02d" % closure
+  return bond_smarts + "%%%02d" % closure
```

Comment:
F strings

Reply:
OK, I got your idea! I would fix it in the next patch. Thank you for your review.